libs/x-content/src/test/java/org/elasticsearch/xcontent/ConstructingObjectParserTests.java

```diff
@@ -817,7 +817,7 @@ public void testRemovalOfField() throws IOException {
             );
             StructRemovalField parse = StructRemovalField.PARSER.parse(parser, null);
 
-            assertWarnings("The field old_name has been removed and is being ignored");
+            assertCriticalWarnings("The field old_name has been removed and is being ignored");
```

Comment:
REST API compatibility still warns at critical level.

Reply:
I guess it is ok? compatibility warns about a feature that was already removed